processor/transformprocessor/internal/common/parser_test.go

```diff
@@ -314,3 +348,7 @@ func floatp(f float64) *float64 {
 func intp(i int64) *int64 {
 	return &i
 }
+
+func boolp(b bool) *Boolean {
+	return (*Boolean)(&b)
+}
```

Comment:
The `strp`, `intp`, and `floatp` functions were moved to a `testhelper` package in `common`.  Please update references and move this new helper there as well

Reply:
Rebased and incorporated these changes; Will rebase again after #10472 is merged, I had to make those changes to get the tests working.

Reply:
Unit tests are fixed.